hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -202,6 +204,9 @@
     protected static final String NULL_AGGREGATOR_IS_NOT_ALLOWED = "Aggregator should not be null!";
     protected static final String NULL_PROJECTION_IS_NOT_ALLOWED = "Projection should not be null!";
 
+    protected static final String MAX_IDLE_TIME_IS_TOO_SMALL = "Parameter maxIdle in seconds representation must be greater or"
```

Comment:
I am not arguing my english is perfect here but I think the message must be "greater **than** or equal **to**"

Reply:
You're right. Fixed. Thanks.